ruby/alias_manager.rb

```diff
@@ -0,0 +1,62 @@
+# ask user for their full name
+# split the string
+# swap between first and last name
+# create a loop that cheks if letter is vowel or consonant
+# 	-if vowel - create a method that finds the next vowel and call it
+# 	-else use .next to replace the consonant.
+continue = true
+new_names = []
+original_names = []
+counter = 0
+vowels = ['a', 'e', 'i', 'o', 'u']
```

Comment:
You vowels and consonants data structure would fit best as local variables within your method.

When working with methods the key is to encapsulate the processes into a meaning space.  In other words, your methods should be holding anything that you will be consistently using and the variables being passed in should be the objects that are subject to change.